gallery/src/styles/NxCard/NxCardPage.tsx

```diff
@@ -103,9 +97,27 @@ const NxCardPage = () =>
               <code className="nx-code">.nx-card__content</code> applied to a
               {' '}<code className="nx-code">&lt;div&gt;</code> wraps
               {' '}<code className="nx-code">.nx-card__call-out</code> and
-              {' '}<code className="nx-code">.nx-card__call-out-text</code> in the alternate row card layout.
+              {' '}<code className="nx-code">.nx-card__text</code> or any other card content.
             </td>
           </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card__call-out</code></td>
+            <td className="nx-cell">
+              Element, typically a child of <code className="nx-code">.nx-card__content</code> in a vertically oriented
```

Comment:
Why does this say "in a vertically oriented card"?  Seem to be used in horizontal cards as well.

Reply:
> Why does this say "in a vertically oriented card"? Seem to be used in horizontal cards as well.

It is used in horizontal cards but it is not a child of `nx-card__content` in horizontal cards. This is one of the breaking changes.

Reply:
> It is used in horizontal cards but it is not a child of nx-card__content in horizontal cards. This is one of the breaking changes.

Ok, I think this wording could be improved then; as it is now it suggests that this element is only used in vertically oriented cards

Reply:
> > It is used in horizontal cards but it is not a child of nx-card__content in horizontal cards. This is one of the breaking changes.
> 
> Ok, I think this wording could be improved then; as it is now it suggests that this element is only used in vertically oriented cards

How's this? https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/9074805fa28bd78b150190e7170a1410c77c4819